Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM mhart/alpine-node:4
+# docker-kassy
+# -----------------------------------------------------------------------------
+# Alpine Linux containerised version of Kassy ready for deployment
+# -----------------------------------------------------------------------------
+# Make sure to map your config.json file into /kassy!
+# see: kassy/doc/integrations/Facebook.md for the file format
+MAINTAINER Matt Hartstonge <matt@mykro.co.nz>
+
+COPY ./ /kassy
+RUN apk --no-cache add \
+        bash \
+        g++ \
+        make \
+        python \
+    && mv /kassy/kassy /usr/bin/kassy \
+    && chmod 744 /usr/bin/kassy \
+    && cd /kassy \
+    && npm install \
+    && apk del \
+        python \
+        make \
+        g++
+
+WORKDIR ["/kassy"]
+VOLUME ["/kassy/modules/"]
+ENTRYPOINT ["kassy"]
+CMD ["--fb"]
```

Comment:
So, no initial switch then?
I'll write docs up tomorrow about how to use it e.t.c.
On Sat, 14 May 2016 at 11:32 PM, Matthew Knox <notifications@github.com>
wrote:

> In Dockerfile
> <https://github.com/concierge/Kassy/pull/157#discussion_r63274901>:
>
> > +        g++ \
> > +        make \
> > +        python \
> > +    && mv /kassy/kassy /usr/bin/kassy \
> > +    && chmod 744 /usr/bin/kassy \
> > +    && cd /kassy \
> > +    && npm install \
> > +    && apk del \
> > +        python \
> > +        make \
> > +        g++
> > +
> > +WORKDIR ["/kassy"]
> > +VOLUME ["/kassy/modules/"]
> > +ENTRYPOINT ["kassy"]
> > +CMD ["--fb"]
>
> Technically test is default anyway as when no commands are provided to the
> commandline that is what it will launch.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/concierge/Kassy/pull/157/files/08df1c7c3da0c19c769677ac5e6dd8ef2f5daabb#r63274901>
>